sdk/src/org.graalvm.nativeimage/src/org/graalvm/nativeimage/c/type/CTypeConversion.java

```diff
@@ -275,4 +275,11 @@ public static CCharPointerPointerHolder toCStrings(CharSequence[] javaStrings) {
     public static ByteBuffer asByteBuffer(PointerBase address, int size) {
         return ImageSingletons.lookup(CTypeConversionSupport.class).asByteBuffer(address, size);
     }
+
+    /**
+     * Provides access to a C pointer for the provided Java byte array.
+     */
+    public static CCharPointerHolder toCBytes(byte[] bytes) {
```

Comment:
Additions to the public API also require the API signature files to be updated. Please make sure that `mx sigtest` passes.

Reply:
The method is added in the `sdk/src/org.graalvm.nativeimage/snapshot.sigtest` file. But it reports many errors from other methods. It seems the `snapshot.sigtest` file is not updated with the source code.